### PR TITLE
Create chat page template

### PR DESCRIPTION
**Pull request by littlestar642**
Solved issue #3

### CODE REVIEW

**Comment by sumedh123** [comment]
@littlestar642 Can you move this page in views? Or just provide a deployment for the same so that I can see it? Some where on gh-pages?

**Comment by littlestar642** [comment]
I have implemented a route to the chat page .Also modified the chat page.

File: views/chat.hbs [hunk 1]
```diff
@@ -1,10 +1,61 @@
...
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.2.13/dist/semantic.min.css">
     <title>Chat: Omeclone</title>
   </head>
-  <body>
-    Chat template where the strangers talk.
+  <style>
+    .seven{
```
Comment (sumedh123) [inline views/chat.hbs]: Why do you want to use custom styling? Semantic UI has great components. Please remove this and do it with semantic classes.

File: views/chat.hbs [hunk 2]
```diff
@@ -1,10 +1,61 @@
...
+
+        <div class="ui action icon input" style="width:100%;">
+            <input type="text" placeholder="What are you sending...">
+            <i class="smiley icon"></i>
+            <button class="ui teal right labeled icon button">
+                <i class="send icon"></i>
+                Send
+            </button>
+        </div>
+
```
Comment (sumedh123) [inline views/chat.hbs]: Remove space.

**Comment by littlestar642** [comment]
Please review the changes requested. @sumedh123

**Comment by sumedh123** [comment]
@littlestar642 Can you sqaush the commits?

**Comment by littlestar642** [comment]
I just squashed the commits.Please review.

**Pull request merged**